package.json

```diff
@@ -7,7 +7,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "parcel --port 3000 src/index.html --out-dir build/debug",
-    "prebuild": "tsc --noEmit",
+    "prebuild": "npm-run-all prebuild:*",
+    "prebuild:clean": "shx rm -rf build",
+    "prebuild:tsc": "tsc --noEmit",
```

Comment:
these are build steps, do we need to add `prebuild`?

```
yarn install --frozen-lockfile
yarn generate
yarn build
```

Reply:
No, all `pre*` are called automatically before the `*` task ([doc reference](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts))

Reply:
thanks